base/include/BrightnessContrastControl.h

```diff
@@ -5,15 +5,16 @@
 class BrightnessContrastControlProps : public ModuleProps
 {
 public:
-	BrightnessContrastControlProps(double _alpha, int _beta) : alpha(_alpha), beta(_beta)
+	BrightnessContrastControlProps(double _contrast, double _brightness, int _bitsPerPixel) : contrast(_contrast), brightness(_brightness), bitsPerPixel(_bitsPerPixel)
```

Comment:
why should bpp (bits per pixel) be a module prop... it is a Pin Property, The brighness/contrast computation remians the same weather the input is 8bpp, 12bpp or 24bpp, it is not different then the chanel (YUV, RGB, etc.)